packages/network-of-terms-catalog/catalog/queries/lookup/rtf-personen.rq

```diff
@@ -8,7 +8,7 @@ CONSTRUCT {
         skos:altLabel ?altLabel ;
         skos:scopeNote ?scopeNote ;
         skos:exactMatch ?exactMatch_uri ;
-        rdfs:seeAlso ?seeAlso_uri .
+        rdfs:seeAlso ?uri .
```

Comment:
Please simply remove `rdfs:seeAlso` completely: if it’s missing, the website https://termennetwerk.netwerkdigitaalerfgoed.nl will simply direct users to the term’s URI.